### ISSUE TITLE
Reflect in box.status() overall server status for DBA to take action if necessary

### ISSUE THREAD

**Issue by kostja** [tarantool/tarantool#71]
Introduce a new field in box.status(), called 'status', in which reflect the current health of the running server.

Rename existing status field to 'role' (master/replica/hot_standby) or come up with a different field name.

If everything is OK, the field needs to contain OK or be empty.
If last snapshot write failed, the field needs to contain "snapshot write failed". If the server failed to allocate memory, so the field needs to reflect it.
If the server failed to open xlog for write, the field needs to state this as well.

In other words, the field only needs to contain state of the server which is relevant for immediate action by the DBA.

It should be possible to reset this field.

**Comment by kostja** [comment]
Impossible to do.
